Package.swift

```diff
@@ -0,0 +1,56 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+	name: "SwiftGit2",
+	platforms: [
+		.macOS(.v10_11),
+		.iOS("9.2"),
+	],
+	products: [
+		.library(
+			name: "SwiftGit2",
+			targets: ["SwiftGit2"]
+		),
+	],
+	dependencies: [
+		.package(url: "https://github.com/Quick/Quick", from: "2.2.0"),
+		.package(url: "https://github.com/Quick/Nimble", from: "8.0.8"),
+		.package(url: "https://github.com/marmelroy/Zip.git", from: "2.0.0"),
+	],
+	targets: [
+		.systemLibrary(
+			name: "Clibgit2",
+			pkgConfig: "libgit2",
+			providers: [
+				.brew(["libgit2"]),
+				.apt(["libgit2-dev"]),
+			]
+		),
```

Comment:
So I've looked into this as well, and it seems that this will indeed successfully build libgit2 using the small shim/wrapper, but only for the architecture checked out by Homebrew on Mac. In other words, this won't work when you want to release a Universal binary with Intel and ARM architecture support. An XCFramework of libgit2 with all architectures might do the job, but I'm not quite there, yet. See #203 for reference.